src/components/MediatedRequestsActivities/MediatedRequestsActivities.js

```diff
@@ -16,38 +15,70 @@ import {
 } from '@folio/stripes/smart-components';
 
 import NavigationMenu from '../NavigationMenu';
-import MediatedRequestsFilters from './components/MediatedRequestsFilters';
+import {
+  MediatedRequestsFilters,
+  MediatedRequestsList,
+} from './components';
 
 import {
   APP_ICON_NAME,
   FILTER_PANE_WIDTH,
+  MEDIATED_REQUESTS_RECORDS_NAME,
   getMediatedRequestsActivitiesUrl,
 } from '../../constants';
 
-const MediatedRequestsActivities = ({ settings }) => {
+const MediatedRequestsActivities = ({
+  querySetter,
+  queryGetter,
+  source,
+  resources,
+  mutator: {
+    resultOffset,
+  },
+  settings,
+}) => {
   const [filterPaneIsVisible, setFilterPaneIsVisible] = useState(true);
 
   const toggleFilterPane = () => {
     setFilterPaneIsVisible(!filterPaneIsVisible);
   };
 
-  const renderResultsFirstMenu = () => {
+  const renderResultsFirstMenu = (filters) => {
     if (filterPaneIsVisible) {
       return null;
     }
 
+    const filterCount = filters.string !== '' ? filters.string.split(',').length : 0;
```

Comment:
Lets avoid using negative scenarios as we discussed:

```suggestion
    const filterCount = filters.string == '' ? 0 : filters.string.split(',').length ;
```

Reply:
Fixed